artifactory.py

```diff
@@ -1502,6 +1594,12 @@ class PureArtifactoryPath(pathlib.PurePath):
 
     parser = _artifactory_flavour
     _flavour = parser  # Compatibility shim for Python < 3.13
+
+    # In Python 3.13, this attribute is accessed by PurePath.glob(), and we need to
+    # override it to behave properly for ArtifactoryPaths with a custom subclass of
+    # glob._Globber.
+    _globber = _ArtifactoryGlobber
```

Comment:
If there's no need for that in python < 3.13 may be we could add condition (just to be safe about that in previous versions)
```suggestion
    if IS_PYTHON_3_13_OR_NEWER:
        _globber = _ArtifactoryGlobber
```

Reply:
That's fair. I debated between conditionalizing this and arbitrarily chose the other way, but I can add in this `if` statement.